### PR TITLE
Add bid service tests

### PR DESCRIPTION
**Pull request by PatrykPodworski**

### CODE REVIEW

File: Test/BidServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+using Xunit;
+
+namespace Test
+{
+    public class BidServiceTests : DatabaseTest
+    {
+        [Fact]
+        public async Task BidCreateAsync_ShouldAddObject()
+        {
+            Prepare();
```
Comment (krzysiekprzekwas) [inline Test/BidServiceTests.cs]: Couldn't we move this to the constructor?

Comment (PatrykPodworski) [reply Test/BidServiceTests.cs]: I do not like this idea, even tho it's xUnit.

Comment (krzysiekprzekwas) [reply Test/BidServiceTests.cs]: Why? You are using this in every test case

Comment (PatrykPodworski) [reply Test/BidServiceTests.cs]: Cause it's not something a constructor should do. And the way xUnit handle test initialization does not convince me.

**Pull request merged**